tests/utils/fixture.ts

```diff
@@ -322,6 +322,9 @@ export async function uploadBlobs(ctx: FixtureTestContext, blobsDir: string) {
   )
 }
 
+const DEFAULT_FLAGS = {
+  serverless_functions_nextjs_durable_cache_disable: true,
```

Comment:
Since I implemented the flag logic to treat a nullish flag as `false` (i.e. don't disable Durable Caching, so that we can actually archive the feature flag someday), dozens of integration tests (which were implicitly mocking runtime `context.flags` to be empty) started failing. This change fixes that, by opting those out of the flag for now.